tests/pytype_test.py

```diff
@@ -25,35 +34,48 @@ def main():
     code, runs = pytype_test(args)
 
     if code:
-        print("--- exit status %d ---" % code)
+        print('--- exit status %d ---' % code)
         sys.exit(code)
     if not runs:
-        print("--- nothing to do; exit 1 ---")
+        print('--- nothing to do; exit 1 ---')
         sys.exit(1)
 
+_PARSE_ONLY_REGEX = r'\s*#\s*parse only\s*'
 
 def load_blacklist():
     filename = os.path.join(os.path.dirname(__file__), "pytype_blacklist.txt")
-    regex = r"^\s*([^\s#]+)\s*(?:#.*)?$"
-
+    regex = r'^\s*([^\s#]+(?:%s)?)\s*(?:#.*)?$' % _PARSE_ONLY_REGEX
     with open(filename) as f:
-        return re.findall(regex, f.read(), flags=re.M)
+        lines = re.findall(regex, f.read(), flags=re.M)
+    blacklist = []
+    parse_only = []
+    parse_only_re = re.compile(_PARSE_ONLY_REGEX)
+    for f in lines:
+        if parse_only_re.search(f):
+            parse_only.append(f.split('#')[0].strip())
```

Comment:
Make the regexp so that you can do

```
match = parse_only_re.match(f)
if match:
  parse_only.append(match.group(1))
```
?

Reply:
Refactored so that the regexes are independent of each other.